config/index.ts

```diff
@@ -81,11 +81,11 @@ export function writeConfig(options: WriteConfigOptions = {}): void {
   }
 }
 
-export function getConfigPath(path?: string): string | null {
-  if (CLIConfiguration.isActive()) {
+export function getConfigPath(useNewConfig = false): string | null {
```

Comment:
This is also _technically_ a breaking change even though it doesn't look like we actually pass the string in anywhere. If possible might be best to find a way to do this without breaking changes. Maybe adding a new method?

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/172/commits/b8c4b88463b52a4faf4448223fe68e58064d52af